contracts/abstracts/ListingManager.sol

```diff
@@ -43,6 +44,14 @@ abstract contract ListingManager is Controlable {
     emit ListingCreated(listingId, tokenContract, tokenId, salePrice, seller);
   }
 
+  function _editListingPrice(uint256 listingId, uint256 newSalePrice) internal returns (bool success) {
+    require(_listings[listingId].salePrice == 0, 'NFT minted');
+    require (_listings[listingId].seller == msg.sender, 'Message Sender is not the seller');
+    // require (_listingId.isListingActive == true, 'Listing not active');
+
+    emit ListingPriceChanged(listingId, newSalePrice);
```

Comment:
You emit the event to broadcast the event with the new price, but you don't actually change the price in the storage, you need to add that part as well